common/models/config-file.js

```diff
@@ -248,10 +246,11 @@ module.exports = function(ConfigFile) {
 
       cb(null, result);
     });
-  }
+  };
 
   function entityBelongsToFacet(name, definition) {
-    return definition && definition.properties && definition.properties.facetName;
+    return definition && definition.properties &&
+      definition.properties.facetName;
```

Comment:
@superkhau ^^^ that does not work, `&&` must be at the end of the line, not at the beginning as in your example.